docs/source/tutorials/AfriSAR_AGB_Maps_1681.ipynb

```diff
@@ -0,0 +1,1381 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    import rioxarray
</pre><p>I believe we need to add <code>rioxarray</code> to requirements.txt </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/140/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>